config/application.yml.default

```diff
@@ -126,6 +126,7 @@ enable_rate_limiting: true
 enable_test_routes: true
 enable_usps_verification: true
 event_disavowal_expiration_hours: 240
+facial_match_general_availability_enabled: true
```

Comment:
(suggestion, non-blocking): prefix with `feature_`? (I'm not actually clear on our feature flag naming conventions, if any.)